edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/identity/UsernameParser.cs

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Core.Identity
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public interface IUsernameParser
```

Comment:
Nit: Suggest renaming to - MqttUsernameParser, or IoTHubIdentityMqttUsernameParser